prow/cmd/branchprotector/protect.go

```diff
@@ -39,11 +38,11 @@ type options struct {
 
 func (o *options) Validate() error {
 	if o.config == "" {
-		return errors.New("empty --config")
+		return fmt.Errorf("empty --config")
```

Comment:
it is just that it was inconsistent all over the code. Some use fmt.Errorf some use errors.New. Since errors was conflicting with errors renamed from Errors I used fmt.Errorf

Reply:
will revert to ease review